view/embed-code.php

```diff
@@ -61,9 +61,9 @@
 <?php endif; ?>
 <script>
 	kWidget.embed({
-		"targetId": <?php echo wp_json_encode($playerId); ?>,
```

Comment:
I've updated the minimal version of the plugin (https://github.com/kaltura/all-in-one-video-pack.wordpress/commit/e612e48cebca0d56af44116e66a90ed7a6eda6bb), we need to keep `wp_json_encode` for VIP certification.